pagination/src/main/java/com/cerner/beadledom/pagination/BeadledomPaginationModule.java

```diff
@@ -0,0 +1,127 @@
+package com.cerner.beadledom.pagination;
+
+import com.cerner.beadledom.pagination.parameters.LimitParameter;
+import com.cerner.beadledom.pagination.parameters.OffsetParameter;
+
+import com.google.inject.AbstractModule;
+import com.google.inject.Provides;
+import com.google.inject.name.Named;
+
+import javax.inject.Singleton;
+
+/**
+ * Module to provide the necessary configurations for using Offset Based Pagination.
+ *
+ * <p>Module provides the configurations needed to use offset based pagination. This includes
+ * the {@link OffsetPaginatedListLinksWriterInterceptor} as well as default values for the limit and
+ * offset fields and their respective field names. The default values for the limit, offset, and
+ * their respective field names can be configured using one of the provided constructors.
+ *
+ * @author Will Pruyn
+ * @since 2.9
+ */
+public class BeadledomPaginationModule extends AbstractModule {
```

Comment:
Rename to BeadledomOffsetPaginationModule

Reply:
Will future pagination live in a separate module? I was under the impression if we added cursor based pagination it would be in this module.

Reply:
Could be shortened to just `OffsetPaginationModule`.